app/Modules/Core/Model/Entity/EventTag.php

```diff
@@ -0,0 +1,82 @@
+<?php declare(strict_types=1);
+
+namespace App\Modules\Core\Model\Entity;
+
+use Nette\Database\Table\ActiveRow;
+
+final class EventTag
+{
+	/** @var Event */
+	private $event;
+
+	/** @var Tag */
+	private $tag;
+
+	/** @var int */
+	private $rate;
+
+	public static function createFromRow(ActiveRow $eventTagRow)
+	{
+		$eventTag = new EventTag();
+		$eventTag
+			->setTag(Tag::createFromRow($eventTagRow->ref('tags', 'tag_id')))
```

Comment:
As Tomáš said, I also prefer construct dependency nowadays, so please get rid of setters completely and create the new entities immutable.